tests/test_auxtel_track_target_and_take_image.py

```diff
@@ -333,9 +353,33 @@ async def setup_mocks(self):
         self.script.latiss.take_object = unittest.mock.AsyncMock(
             side_effect=self.take_object_side_effect
         )
+        self._ataos_evt_correction_enabled = types.SimpleNamespace(
```

Comment:
How is it local if it's used in line 381?

Reply:
"private" means internal to the class... So methods of the class can call certainly call it, but someone using the class should not. 

It is basically a way of saying, don't use this outside the class.. which might be a bit exaggerated here since this is a test case class, not something people will be reusing..

Reply:
ah ok... I thought it meant it should only be used inside the method... not the class.